### PR TITLE
Prefer Jaguar over Toit.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: vscode/src/extension.ts [hunk 1]
```diff
@@ -77,45 +75,34 @@ async function missingJagSetupPrompt(jagExec: string) : Promise<boolean> {
...
-  const action = await Window.showErrorMessage(message, installAction, settingsAction);
-  if (action === installAction) {
-    env.openExternal(Uri.parse("https://docs.toit.io/getstarted/installation"));
-  } else if (action === settingsAction) {
-    Commands.executeCommand( "workbench.action.openSettings", "toit.path" );
+async function missingLspPrompt() {
+  const installJagAction = "Install Jaguar";
+  let message = "Could not find the `jag` or `toit` executable. ";
+  message += "Please make sure `jag` or `toit` is installed or set the jag.path ";
+  message += "(resp. toit.path) setting to the executable and reload the window to activate the extension";
```
Comment (kasperl) [inline vscode/src/extension.ts:82]:
```suggestion
  message += "(or toit.path) setting point to the executable and reload the window to activate the extension";
```

File: vscode/src/extension.ts [hunk 2]
```diff
@@ -77,45 +75,34 @@ async function missingJagSetupPrompt(jagExec: string) : Promise<boolean> {
...
-  message += ". Please make sure `toit` is installed and set the toit.path setting to the executable (reload the window to activate the extension";
-  const action = await Window.showErrorMessage(message, installAction, settingsAction);
-  if (action === installAction) {
-    env.openExternal(Uri.parse("https://docs.toit.io/getstarted/installation"));
-  } else if (action === settingsAction) {
-    Commands.executeCommand( "workbench.action.openSettings", "toit.path" );
+async function missingLspPrompt() {
+  const installJagAction = "Install Jaguar";
+  let message = "Could not find the `jag` or `toit` executable. ";
+  message += "Please make sure `jag` or `toit` is installed or set the jag.path ";
```
Comment (kasperl) [inline vscode/src/extension.ts:81]:
```suggestion
  message += "Please make sure `jag` or `toit` is installed or let the jag.path ";
```

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: vscode/src/extension.ts (hunk 2 above)
Comment (floitsch) [reply vscode/src/extension.ts]: Changed to:
"Please make sure `jag` or `toit` is installed. If not in PATH, update the 'jag.path' (or 'toit.path') setting. Then reload the window."

File: vscode/src/extension.ts (hunk 1 above)
Comment (floitsch) [reply vscode/src/extension.ts]: See above.

**Pull request merged**